webapp/package.json

```diff
@@ -44,7 +44,8 @@
     ],
     "moduleNameMapper": {
       "^@/(.*)$": "<rootDir>/src/$1",
-      "^~/(.*)$": "<rootDir>/$1"
+      "^~/(.*)$": "<rootDir>/$1",
+      "\\.(css|less)$": "identity-obj-proxy"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-01T21:30:27Z" title="Tuesday, October 1st 2019, 11:30:27 pm +02:00">Oct 1, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -44,7 +44,8 @@
     ],
     "moduleNameMapper": {
       "^@/(.*)$": "<rootDir>/src/$1",
-      "^~/(.*)$": "<rootDir>/$1"
+      "^~/(.*)$": "<rootDir>/$1",
+      "\\.(css|less)$": "identity-obj-proxy"
```


What is that? :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-01T21:53:16Z" title="Tuesday, October 1st 2019, 11:53:16 pm +02:00">Oct 1, 2019</time>_

---


it is for mocking CSS modules in Jest
so we are importing the CSS for cropperjs, but it breaks the tests, so we need to mock it
https://jestjs.io/docs/en/webpack#mocking-css-modules